pkg/tui/tui.go

```diff
@@ -6,6 +6,13 @@ import (
 
 	"github.com/amlweems/timebomb/pkg/engine"
 	tea "github.com/charmbracelet/bubbletea"
+	"github.com/muesli/termenv"
+)
+
+var (
+	color = termenv.EnvColorProfile().Color
+	red = termenv.Style{}.Foreground(color("9")).Styled
+	blue = termenv.Style{}.Foreground(color("12")).Styled
```

Comment:
For some reason these colors are not working for me locally, do you know if I need some special terminal config?

Reply:
I'm not sure how it works, but when I connect locally or from my corp laptop, both running desktop Linux, I see the following entry:
```
2024/02/08 18:12:27 INFO eze connect 127.0.0.1:42388 false [cvr] xterm-256color 275 73
```

I imagine `256color` indicates that it supports these colours, maybe your computer is running something different?

But given that the text & colour is generated server-side, I'm not sure how to handle those cases.